showmore_dyn/templates/settings.tpl

```diff
@@ -0,0 +1,11 @@
+<div class="settings-block">
+	<h3 class="settings-heading">{{$title}}</h3>
+	<div class='field noedit'>
```

Comment:
There are sub-templates for input fields, please look into the expected data structure: friendica.mrpetovan.com/help/smarty3-templates#field_input.tpl

You can check the `blackout` addon for a usage of the `field_input.tpl` sub-template.